Client.xcodeproj/project.pbxproj

```diff
@@ -693,7 +693,6 @@
 		9636D92E27F9E5D900771F5E /* GleanPlumbMessage.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9636D92D27F9E5D900771F5E /* GleanPlumbMessage.swift */; };
 		9638332327E14ACC0011EEFC /* AccessibilityIdentifiers.swift in Sources */ = {isa = PBXBuildFile; fileRef = C84266742728462900382274 /* AccessibilityIdentifiers.swift */; };
 		963A879A29AFE89B006DEE34 /* SensitiveHostingController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 963A879929AFE89B006DEE34 /* SensitiveHostingController.swift */; };
-		96441BD629DF823D00FE041F /* MockAppAuthenticator.swift in Sources */ = {isa = PBXBuildFile; fileRef = 96441BD529DF823D00FE041F /* MockAppAuthenticator.swift */; };
```

Comment:
We need this file to the `ClientTest` suite

Reply:
oof, got removed during one of my cleanup
added it back